site/en/100/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: '#100CoolWebMoments'
+description: 'Take a stroll down memory lane and celebrate #100CoolWebMoments since Chrome’s first release.'
+thumbnail: ''
```

Comment:
Please add thumbnail, as the page card will not render without it in an index page.

Reply:
Added!